tests/cli/test_cli_options.py

```diff
@@ -726,6 +737,7 @@ def main(args):
                                   reg_replace(r', \[[_|,|1|A-Z]*\]', ', []'),
                                   reg_replace(r', \[0[xX][0-9a-fA-F]+:[_|,|1|A-Z]*\]', ', []'),
                                   reg_replace(r'   \[0[xX][0-9a-fA-F]+:[_|,|1|A-Z]*\]', '   []'),
+                                  reg_replace(r'^[ \t]*[_A-Za-z][_A-Za-z0-9]*::', ''),
```

Comment:
This should've resolved the CI build failure in 'tests/cli/output/92.txt'...

Reply:
fine work. push it up